_components/accordions.md

```diff
@@ -4,9 +4,153 @@ type: component
 title: Accordions
 ---
 
-<div class="preview">
-  <!-- Add HTML markup for example here -->
-  <img src="{{ site.baseurl }}/assets/img/static/Accordions_UI_v1.png">
+<h3>Bordered Accordion</h3>
+
+<div class="code-sample">
+  <ul id="accordion-menu">
```

Comment:
@carodew and I just chatted about this.

We think there are several legitimate uses of multiple accordions per page. (e.g. Accordion nav and accordion in the content.) We also think that the library should lean towards accommodating lots of different uses for a given component (think that's been the strategy so far). So the UX team is all for making it possible to have multiple accordions per page. 

cc: @juliaelman and @mollieru